workflows/rnaseq-ref-index/get-refs.sh

```diff
@@ -17,6 +19,16 @@ aws s3 sync fasta $s3_base/fasta
 
 # get annotation files & sync
 wget -N -P annotation -i annotation_ref_urls.txt
-$rdocker Rscript build_tx2gene_mito.R
-aws s3 sync annotation $s3_base/annotation
 
+# running without docker for now due to docker error
+# $rdocker Rscript make_pre_mrna_fasta.R
+
+Rscript make_pre_mrna_fasta.R
+gzip annotation/*.gtf
+
+# the sync with fasta would need to happen here if we are to remove the
+# previous step of combining cdna & ncrna fasta files
+#aws s3 sync fasta $s3_base/fasta
```

Comment:
I think we do want to sync this folder too now that the fasta files go here?
```suggestion
aws s3 sync fasta $s3_base/fasta
```